organizing/speakers-guide.md

```diff
@@ -1,9 +1,62 @@
 
-* Workshop
\ No newline at end of file
+* **Please send us the following 1 month before the meetup:**
+    * Talk/workshop title and description
+    * short bio
+    * twitter handle (if you would like to link our tweets with your account)
+    * Example: https://www.meetup.com/PyLadies-Berlin/events/260312181/
+    * If there are resources that might be helpful for attendees, they can be included in the description.
+
+* **What we also need to know in a timely manner:**
+    * What kind of coaching would you like from us, if any?
+    * Do you need anything more than HDMI cable?
+
+* **Meetup day:** It is best to come 15-30 mins earlier to check that everything works fine with your laptop. If you prefer to use our laptop, you need to send the talk to us beforehand.
+
+* **Resources for an amazing talk:** https://github.com/PyLadiesBerlin/resources#public-speaking---call-for-proposals---cfp
+
+
+## Additional workshop guidelines:
+
+* **Session format and time:** 
+    * Keep talk and workshop apart, if possible. 
+    * We suggest the talk to be 30 mins or less (time is aligned with conference talks, so you could take the talk to a conference as well); the hands-on part can be an hour.
+    * Before your part, we have an intro and there might be a company short talk first.
+    * After the talk, take a 15-20 mins break for networking and setup. People are more comfortable to talk after the talk, so it is a good time to let them mingle. 
+    * After networking allocate up to 10 mins for setup (for the people that had problems to set up at home).
+
+* **Workshop format:**
+    * Please make sure that the hands-on part is possible to follow as a beginner.
+    * Active coding facilitates learning - and is more fun than only typing code from the screen or copy/pasting.
+    * From our experience a self-paced format where attendees take their own time with coaches to support them is best for mixed level audience. 
+    * We	highly recommend to pair people as part of the workshop.
+    * We have also had a lot of success with an interactive format where attendees had to choose cards with workshop-related topics which they then had to explain to each other. Example: https://github.com/PyLadiesBerlin/resources/blob/master/workshops/23_Data_Wrangling_Pandas/README.md
+    * If it is a code along format, consider a catch-up point giving people time to catch up if they are left behind.
+    * If there are exercises to complete, it is generally a useful practice to have already solved shortcut steps that the attendees can take if they get stuck (e.g. folder with solved steps - example: https://github.com/mei-li/multiling-chat)
+
+* **Workshop coaches:** Usually, if the workshop is self-paced, it is important to have people (coaches) helping on the spot. Having a couple of coaches for technical issues is always helpful. Let us know, if coaches are needed.
```

Comment:
maybe here it is worth to say that the audience can be up to 40 people so more coaches might be needed